static/js/common/DropdownMenu.jsx

```diff
@@ -44,10 +44,10 @@ const DropdownMenuItemWithIcon = ({icon, textEn, textHe}) => {
   );
 }
 
-const DropdownMenu = ({children, buttonComponent}) => {
+const DropdownMenu = ({children, buttonComponent, positioningClass}) => {
     /**
      * buttonComponent is a React component for the opening/closing of a button.
```

Comment:
Please edit this docstring, adding in an explanation about the `positioningClass`